hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/overviewCard/overviewCard.tsx

```diff
@@ -46,18 +46,31 @@ const defaultProps = {
 
 interface IOverviewCardWrapperProps {
   linkToUrl: string;
+  title:string
 }
 
 class OverviewCardWrapper extends React.Component<IOverviewCardWrapperProps> {
   render() {
     const {linkToUrl, children} = this.props;
-    if (linkToUrl) {
+    if (linkToUrl ) {
+      if(linkToUrl === '/Om'){
```

Comment:
Can we shorten this check? Instead of spanning two lines can you try to make the check as:
```
if (linkToUrl){
    return (
        <Link to={{
            pathname: linkToUrl
            state: { linkToUrl === "/Om" ? { activeTab: .... } : undefined }
            }}
        />
    )
}
```

Could you check this out?

Reply:
Rest looks good to me

Reply:
@devabhishekpal 
It should not allow to enter and use  <Link > tag so gave additional condition to mimimize computation time.

Reply:
Okay, looks good to me